app/scripts/modules/core/src/delivery/delivery.states.ts

```diff
@@ -67,7 +67,7 @@ module(DELIVERY_STATES, [
     url: `?${stateConfigProvider.paramsToQuery(filterModelConfig)}`,
     views: {
       'pipelines': {
-        template: '<executions application="$resolve.app"></executions>',
+        component: Executions, $type: 'react',
```

Comment:
separate lines here, or just inline the whole thing, i.e. `{ component: Executions, $type: 'react' }`?